src/main/java/ua/edu/internship/interview/service/dto/user/question/UserQuestionDto.java

```diff
@@ -0,0 +1,15 @@
+package ua.edu.internship.interview.service.dto.user.question;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import ua.edu.internship.interview.service.dto.skill.SkillDto;
+
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper=true)
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '=' is not preceded with whitespace.
<!-- __reviewdog__:ChA0OWI0N2EzNGVlMTg3NDlhEgpjaGVja3N0eWxl -->